migrate_pim/index.rst

```diff
@@ -123,6 +124,7 @@ Here are the migration guides:
 * `From v1.1 to v1.2`_
 * `From v1.0 to v1.1`_
 
+.. _From v2.1 to v2.2: https://github.com/akeneo/pim-community-standard/blob/master/UPGRADE-2.2.md
```

Comment:
You'll need to wait for 2.2 to be merged into master for this link to work and have a green CI.

It is important that is points to the master branch and not the 2.2, because the 2.2 branch will disappear shortly after 2.3 release, not the master branch.